javascript/dwa-starter-vue/.tbd-example.json

```
@@ -0,0 +1,11 @@
+{
+  "name": "DWA Starter Vue",
+  "tests": {
+    "pre": [
+      "npm install -g pnpm", 
+      "pnpm install", 
+      "npx playwright install --with-deps"
+    ],
+    "command": "pnpm run build && pnpm run test:unit && pnpm run test:e2e"
```

Please put the build step in the pre-test commands and break the commands up:

```suggestion
    "pre": [
      "npm install -g pnpm", 
      "pnpm install", 
      "npx playwright install --with-deps",
      "pnpm run build",
    ],
    "commands": ["pnpm run test:unit", "pnpm run test:e2e"]
```

---

cool fixed now

---

I believe the key needs to be `commands` if you're providing a list.